analyzer/analyzer.go

```diff
@@ -3,187 +3,82 @@ package analyzer
 import (
 	"context"
 	"io"
-	"os"
-	"sync"
 	"sync/atomic"
+	"time"
 
-	"github.com/brimdata/brimcap/ztail"
-	"github.com/brimdata/zed/compiler"
-	"github.com/brimdata/zed/compiler/ast"
-	"github.com/brimdata/zed/driver"
 	"github.com/brimdata/zed/zio"
-	"github.com/brimdata/zed/zng"
-	"github.com/brimdata/zed/zson"
+	"golang.org/x/sync/errgroup"
 )
 
-type Interface interface {
-	zio.ReadCloser
-	BytesRead() int64
-	RecordsRead() int64
-	WarningHandler(zio.Warner)
+type Stats struct {
+	BytesRead      int64
+	RecordsWritten int64
 }
 
-type analyzer struct {
-	cancel   context.CancelFunc
-	config   Config
-	ctx      context.Context
-	once     sync.Once
-	procErr  error
-	procDone chan struct{}
-	reader   *readCounter
-	records  int64
-	tailer   *ztail.Tailer
-	warner   zio.Warner
-	workdir  string
-	zctx     *zson.Context
-	zreader  zio.Reader
+type Driver interface {
```

Comment:
Nit: This thing doesn't really feel like a driver. It's more like a display or a UI, so maybe call it `Display` or `UI`?

Reply:
Sure!